pymilo/pymilo_obj.py

```diff
@@ -37,7 +37,7 @@ def save(self, file_adr):
         Save model in a file.
 
         :param file_adr: file address
-        :type file_adr: str
+        :type file_adr: FileDescriptorOrPath
```

Comment:
Is it true?!

Reply:
Technically yes:
```pycon
FileDescriptorOrPath: TypeAlias = int | StrOrBytesPath
```

but since path creating function like `os.path.join` return `str`, I will rollback to `str`.